src/makeRobot.js

```diff
@@ -38,7 +38,95 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
-}
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    goForward(step) {
+      if (step === undefined) {
+        this.coords.y += 1;
+      }
+
+      if (step < 0) {
+        return this;
+      }
+
+      if (step > 0) {
+        this.coords.y += step;
+      }
+
+      return this;
+    },
+
+    goBack(step) {
+      if (step === undefined) {
+        this.coords.y -= 1;
+      }
+
+      if (step < 0) {
+        return this;
+      }
+
+      if (step > 0) {
+        this.coords.y -= step;
+      }
+
+      return this;
+    },
+
+    goRight(step) {
+      if (step === undefined) {
+        this.coords.x += 1;
+      }
+
+      if (step < 0) {
+        return this;
+      }
+
+      if (step > 0) {
+        this.coords.x += step;
+      }
+
+      return this;
+    },
+
+    goLeft(step) {
+      if (step === undefined) {
+        this.coords.x -= 1;
+      }
+
+      if (step < 0) {
+        return this;
+      }
+
+      if (step > 0) {
+        this.coords.x -= step;
+      }
+
+      return this;
+    },
+
+    evacuate() {
+      this.coords.x = 1400;
+      this.coords.y = 500;
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version:`
+      + ` ${this.version}, wheels: ${this.wheels}`;
```

Comment:
it would be better to divide them into groups rather than by the number of characters
```suggestion
      return `name: ${this.name}, `
        + `chip version: ${this.version}, `
        + `wheels: ${this.wheels}`;
```